tests/test_models.py

```diff
@@ -76,16 +76,25 @@ def test_validate_fullname_empty(self):
         with assert_raises(ValidationValueError):
             self.user.save()
 
-    def test_validate_social_personal_empty(self):
-        self.user.social = {'personal': ''}
+    def test_validate_social_profile_websites_empty(self):
+        self.user.social = {'profileWebsites': []}
```

Comment:
@billyhunt are we just ensuring that the call to `User#save` does not raise anything? Do we want to assert that the `User.social` has what we'd expect?